packages/node_modules/@ciscospark/plugin-conversation/src/conversation.js

```diff
@@ -469,10 +469,16 @@ const Conversation = SparkPlugin.extend({
   /**
    * Creates a ShareActivty for the specified conversation
    * @param {Object} conversation
+   * @param {Object} activity
    * @returns {ShareActivty}
    */
-  makeShare(conversation) {
-    return ShareActivity.create(conversation, null, this.spark);
+  makeShare(conversation, activity) {
```

Comment:
Fine. Make sure you have tests around this in the web client because we'll need to revisit and introduce breaking changes to get `ShareActivity` cleaned up.